pages/common/chsh.md

```diff
@@ -3,6 +3,14 @@
 > Change the user's login shell.
 > More information: <https://manned.org/chsh>.
 
-- Change the user's login shell:
+- Change the current user's login shell interactively:
 
-`chsh -s {{path/to/shell_binary}} {{username}}`
+`chsh`
+
+- Change the login shell of `user`:
+
+`chsh --shell {{/bin/zsh}} {{user}}`
```

Comment:
I think this is the correct example description
```suggestion
- Change the login shell to Zsh of a specific user:

`chsh --shell {{/bin/zsh}} {{username}}`
```

Reply:
it should be `for` instead of `of`

Reply:
```suggestion
- Change the login shell for a given user:

`chsh --shell {{/bin/zsh}} {{user}}`
```